lib/features/sks-menu/presentation/sks_menu_screen.dart

```diff
@@ -52,13 +64,19 @@ class SksMenuView extends ConsumerWidget {
 }
 
 class _SksMenuView extends StatelessWidget {
-  const _SksMenuView(this.sksMenuData, this.appBarPopTitle);
+  const _SksMenuView({
+    required this.sksMenuData,
+    this.appBarPopTitle,
+    required this.isLastMenuButtonClicked,
+  });
 
   final SksMenuResponse sksMenuData;
   final String? appBarPopTitle;
+  final bool isLastMenuButtonClicked;
+
   @override
   Widget build(BuildContext context) {
-    if (!sksMenuData.isMenuOnline) {
+    if (!isLastMenuButtonClicked && !sksMenuData.isMenuOnline) {
```

Comment:
see here, we don't show the animation if the menu is online anyway, so the code from above can be deleted:
```diff
- if (sksMenuData.isMenuOnline && isLastMenuButtonClicked.value) {
-        isLastMenuButtonClicked.value = false;
-      }
```

Reply:
unless I'm missing something

Reply:
i think your right that it's unnecessary, I deleted it